FMI-CROSS-CHECK-RULES.md

```diff
@@ -94,10 +67,7 @@ New FMUs may appear in new versions of the set of Reference FMUs.
 
 - Changes to the Reference FMUs will automatically require a new version.
```

Comment:
why?  and of new version of what? the XC rules? ??

Reply:
Fixed (requires a new version of the reference FMUs - this will likely be in line with what happens in that repo - for @t-sommer to propose exact mechanism)